web/website/content/faq.md

```diff
@@ -6,11 +6,15 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL _now_?" >}}
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+PRQL builds powerful SQL queries today. In addition, we are focusing on several
+integrations. Here's how you can use PRQL:
 
+- **Do real work with PRQL** Prototype your PRQL queries in the
+  [Playground](https://prql-lang.org/playground/) or the
+  [VS Code extension](https://marketplace.visualstudio.com/items?itemName=PRQL-lang.prql-vscode)
+  and copy/paste the resulting SQL into your database.
 - **[dbt-prql](https://github.com/prql/dbt-prql)** allows writing PRQL in
```

Comment:
Unfortunately we could probably remove this from this section now, RIP. Maybe we replace with `prqlr` @eitsupi ?

Or we add a bullet about `prqlc`?

Or `prql-query` @snth ?

(No need to resolve all these before merging)

Reply:
See above

Reply:
```suggestion
```

Reply:
@aljazerzen Hmmm... I had meant to close this and replace with my tweak-faq-2 branch. https://github.com/richb-hanover/prql/blob/tweak-faq-2/web/website/content/faq.md

Let's continue the conversation (about what to do about dbt-prql) over there.